Bookstore: styling

ragangithub opened the pull request:
- Added CSS to style the website .

ragangithub committed:
Add styling

souad988 commented on src/components/BookItem.js:
```diff
@@ -2,24 +2,54 @@ import React from 'react';
-      <li>{itemProp.author}</li>
+      <li className="book-cards">
+        <div className="book-info">
+          <p className="category">{itemProp.category}</p>
```
Greate work done so far, I would just mention that the category is missing in you cards , you can add a static values to make your design much with the template , also please reduce the vertical space between the titles and buttons refer to the images below 

![image (19)](https://github.com/ragangithub/bookstore-react/assets/59707859/89f9adea-f59a-4e00-ae74-53de9a7e6c11)
![image (24)](https://github.com/ragangithub/bookstore-react/assets/59707859/b83a6593-a65b-4a14-9157-6153fa78616e)

souad988 commented on src/components/InputBook.js:
```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
-      </form>
+      <hr className="form-hr" />
+      <div className="add-book">
+        <h3>ADD NEW BOOK</h3>
```
Kindly use the same fonts and colors used in the template the title and divider looks diffrent here, also increase left marging of your component 

![image (18)](https://github.com/ragangithub/bookstore-react/assets/59707859/4a0e917f-f739-448f-9021-d5c5d7fe63ba)
![image (23)](https://github.com/ragangithub/bookstore-react/assets/59707859/b5c047db-e2ee-4767-acf0-11640e8b02ce)

souad988 commented on src/components/Navbar.js:
```diff
@@ -1,20 +1,27 @@
+          </li>
+        </ul>
+      </div>
+      <div className={styles.profile}>
```
Well done by adding the profile icon but please do your best to make it look like the design in terms of size and color 

![image (21)](https://github.com/ragangithub/bookstore-react/assets/59707859/fcb93711-23fd-4219-8095-184284ec3ebc)

![image (26)](https://github.com/ragangithub/bookstore-react/assets/59707859/b4829961-15ca-4103-b214-7b485cfe0791)

souad988 commented on src/components/InputBook.js:
```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
+      <hr className="form-hr" />
+      <div className="add-book">
+        <h3>ADD NEW BOOK</h3>
+        <form action="submit" className="submit">
```
Please this form should take the full width of the ùain section just like it's upper component and inputs should fill the space 

![image (16)](https://github.com/ragangithub/bookstore-react/assets/59707859/500f9a20-2a25-4ee8-9eba-d1f13a941936)
![image (17)](https://github.com/ragangithub/bookstore-react/assets/59707859/72c7f7ac-cebc-4f61-88ca-cca4d3d95834)

souad988 reviewed (changes requested):
Hi @ragangithub  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


![242722065-db428b0c-8dfc-49c4-a156-42f227e32868](https://github.com/ragangithub/bookstore-react/assets/59707859/898c39e6-6fc3-4918-a9e3-94095114ddcd)


## Highlights
Good file structure
No inline styling
Good class names

## Required Changes ♻️
_Check the comments under the review._
_ Also please change the background color as it looks darker than the design , and increase margin of your main section to much with the template 
![image (16)](https://github.com/ragangithub/bookstore-react/assets/59707859/a61e9a04-4adb-4cda-8293-466b88fc20f5)
![image (17)](https://github.com/ragangithub/bookstore-react/assets/59707859/ef2e3cd5-0ee1-4f22-96c7-182182b76464)

 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub committed:
Correct styling

ragangithub committed:
Fix linter error

ragangithub commented:
- Changed the background color  and increased the margin of the main section to much with the template
- Included category and also reduced  the vertical space between the titles and buttons 
- Used the same fonts and colors used in the template and corrected the divider 
- Increased the size of the profile icon and used the same color as in the design
- Form takes the full width

binhussen reviewed (approved):
Hi @ragangithub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- Beautiful UI ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
